.palantir/revapi.yml

```diff
@@ -968,6 +968,9 @@ acceptedBreaks:
     - code: "java.class.removed"
       old: "class org.apache.iceberg.rest.requests.UpdateTableRequest.Builder"
       justification: "Removing deprecated code"
+    - code: "java.class.removed"
+      old: "enum org.apache.iceberg.BaseMetastoreTableOperations.CommitStatus"
```

Comment:
I don't think we can break this API / removing this class without deprecating it first. See https://iceberg.apache.org/contribute/#deprecation-notices